core/backend/src/BackendLoggerCategory.ts

```diff
@@ -60,8 +60,9 @@ export enum BackendLoggerCategory {
 
   /** The logger category used by the following classes:
    * - IModelTileRpcImpl (Tile Uploading)
+   * - TileStorage (tile upload/download)
    */
-  IModelTileUpload = "core-backend.IModelTileUpload",
+  IModelTileStorage = "core-backend.IModelTileStorage",
```

Comment:
This is a breaking change to a `@public` API.

Reply:
Good point, I didn't notice our logger categories were public. I'll keep the old one and add another one then.